### DISCUSSION TITLE
routine load 处于running状态，但一直不执行

### DISCUSSION THREAD

**Discussion by smile28616** [apache/doris#16291] [A - General / Q&A]
kafka里有数据，看着有两个问题：
1.看着fe执行调度好慢，routine load创建了两三分钟才开始创建task调度be
2.如图所示
![image](https://user-images.githubusercontent.com/2900393/215731007-f9367851-ca93-41b0-a0a1-e1a2f1740bee.png)

3.fe日志
![image](https://user-images.githubusercontent.com/2900393/215731185-9775c295-6c8f-43e8-a9ad-38845db54cb9.png)

4.be日志
![image](https://user-images.githubusercontent.com/2900393/215731339-180fa7b6-1957-40d3-b180-7690035c254d.png)

重启fe之后 lag好像是能获取的
![image](https://user-images.githubusercontent.com/2900393/215747677-8565003c-a1f6-4577-97a2-6171950d8a81.png)

**Comment by futurexiong** [comment]
同问，本地docker 1FE 1BE也有这种情况，失败过多被pause之后，resume不起作用。只能stop重新create
